app/views/pages/success_stories.html.erb

```diff
@@ -1,45 +1,115 @@
+          <div class="success-stories-name">- Victor Molina, US Marine Corps veteran</div>
+          <div class="success-stories-accounts--right">
+            <a href="https://github.com/molina1775">GitHub</a>
+            <a href="https://twitter.com/livestocode">Twitter</a>
           </div>
         </div>
+        <div class="card-action center hide-on-med-and-up">
+          <div class="success-stories-name">- Victor Molina, US Marine Corps veteran</div>
+          <a href="https://github.com/molina1775">GitHub</a>
+          <a href="https://twitter.com/livestocode">Twitter</a>
+        </div>
+      </div>
+
+      <div class="valign hide-on-small-only">
+        <a href="https://twitter.com/livestocode">
+          <img class="circle z-depth-3 success-stories-photo success-stories-photo--right" src="https://s3-us-west-2.amazonaws.com/operationcode/victor_molina.jpg" alt="Victor Molina" />
+        </a>
+      </div>
+    </div>
 
-        <hr>
+  </div>
+
+  <div class="row">
+
+    <div class="card valign-wrapper">
+      <div class="valign hide-on-small-only">
+        <a href="https://operation-code.slack.com/messages/0311-code">
+          <img class="circle z-depth-3 success-stories-photo success-stories-photo--left" src="https://s3-us-west-2.amazonaws.com/operationcode/jameel_matin.png" alt="Jameel Matin" />
+        </a>
+      </div>
+      <div class="card-stacked">
+            <div class="card-content light">
+        <p>
+          "I enlisted in the Marine Corps Infantry to serve and win our country's wars. After my service, I searched for a new career. I knew I wanted to be part of the tip of the spear in an industry and I realized the tech industry is leading from the front. <b>I Googled around and found Operation Code, a non-profit which helps veterans like me break into the tech industry. A couple weeks later they found a way to get me into GitHub's <a href="http://codeconf.com/">CodeConf LA</a></b>. There, I found <b><a href="http://sabio.la/">Sabio</a></b>, a local code school which accepts the New GI Bill. Today, I am in the middle of a <b><a href="http://vetshack.us/">hackathon</a></b> sponsored by Sabio in an effort to build a website to help folks in the military be successful by the time they get out. Operation Code not only paved a route for me, they also showed me what I can build with my imagination. <b>Imagine what it can do for other veterans</b>."</b>
+        </p>
+      </div>
+      <div class="card-action left-align">
+        <div class="success-stories-name">- Jameel Matin, US Marine Corps veteran</span>
+      </div>
+      </div>
+
+      
+    </div>
+
+  </div>
+
+  <div class="row center">
+    <a class="btn btn-large btn-action" href="http://operation-code.slack.com/messages/@david" role="button">Submit Your Story &raquo;</a>
+
+    <a class="btn btn-large btn-action" href="/square" role="button">Donate now &raquo;</a>
+  </div>
+
+
+</div>
+
+<hr>
 
   <!-- <div class="row center">
```

Comment:
Will do - wasn't sure if @davidmolina was wanting to keep those for some reason (possible candidates for future vets to add here). Either way, it'll be in the VCS history so we can grab it again.